### DISCUSSION TITLE
How to set drainage boundary conditions

### DISCUSSION THREAD

**Discussion by PEI0214** [idaholab/moose#27013] [Q&A Modules: Porous Flow]
Hi，
In porous fluids, I see that simply setting porepressure=0 in the BCs allows for drainage.
So how to set the boundary conditions to achieve first drainage, then no drainage? Is how to set the control drain switch？

Thanks for your help.

**Comment by cpgr** [comment]
You can use the [Controls] system to turn BC's on/off according to some criteria.

**Reply by PEI0214** [reply]
Thank you very much for your answer. I think I know what to do.
